RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -246,7 +246,7 @@ void handlePostAdventure()
 				buffMaintain($effect[Bounty of Renenutet], 10, 1, 10);
 			}
 
-			if (my_level() < 13 && my_level() > 3 && !get_property("auto_needLegs").to_boolean() && (!($locations[Hippy Camp, The Outskirts Of Cobb\'s Knob] contains my_location()) || have_skill($skill[More Legs])))
+			if (my_level() < 13 && my_level() > 3 && !get_property("auto_needLegs").to_boolean() && (!($locations[Hippy Camp, The Outskirts Of Cobb\'s Knob, The Smut Orc Logging Camp] contains my_location()) || have_skill($skill[More Legs])))
```

Comment:
This was supposed to make us not run ML when doing the Orcs but I realised it does nothing as we're guaranteed to have the More Legs upgrade by the time we start that quest anyway. Can be ignored & I'll make a change on another PR.

Reply:
There is a section in auto_pre_adv that handles governing ML in areas we need low ML. Does that not work in Ed?

Reply:
It does work, I've seen it turn off the MCD and unequip the astral belt. It is what prompted me to attempt to stop it casting Blessing of Serqet when doing the Smut Orcs as I noticed it still had 20 ML from that & was re-applying it after it ran out. Hence this (bad) attempt.